Changelog.md

```diff
@@ -5,6 +5,54 @@ All notable Changes to the Julia package `Manopt.jl` will be documented in this
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.40] – 02/09/2023
```

Comment:
```suggestion
## [0.4.40] – 24/10/2023
```
BTW, I generally prefer YYYY-MM-DD format -- it's less confusing.

Reply:
Oh, too much copy paste. Thanks for noticing. Well, this is the good European dd/mm/yyyy (but I see where Americans can confuse this ;)